src/cli/broker.toit

```diff
@@ -649,7 +658,8 @@ class Broker:
   device-for --id/uuid.Uuid -> DeviceDetailed:
     devices := broker-connection_.get-devices --device-ids=[id]
     if devices.is-empty:
-      ui_.abort "Device $id not found."
+      short := device-short-strings_[id]
+      ui_.abort "Device $short not found."
```

Comment:
Not found sounds a bit like our implementation is wrong. If the user specified the wrong device name or id, maybe we should tell them that it is unknown or non-existing?

Reply:
done.